lib/cavendish/cli.rb

```diff
@@ -38,6 +38,7 @@ def create_command_steps
         Cavendish::Commands::AddCiConfig,
         Cavendish::Commands::AddReadme,
         Cavendish::Commands::ConfigureGit,
+        Cavendish::Commands::AddEASConfiguration,
```

Comment:
I'd put this before the `ConfigureGit` command, because the latter makes an initial commit with everything included until that point, and thus every file created afterwards will be shown as a non-staged change on the user's local git repository.

I made a fix for the other commands in [this commit](https://github.com/platanus/cavendish/pull/26/commits/4f22ab7dca24e9f19c1dd8b8e6b29ecccfc8f356)

Reply:
Making a rebase of the branch, incorporating from master the changes made in https://github.com/platanus/cavendish/pull/26 fixed this :ok_hand: